jvm/workbookapp/src/main/kotlin/org/wycliffeassociates/otter/jvm/workbookapp/ui/screens/chunking/Consume.kt

```diff
@@ -0,0 +1,108 @@
+ */
+package org.wycliffeassociates.otter.jvm.workbookapp.ui.screens.chunking
+
+import com.github.thomasnield.rxkotlinfx.observeOnFx
+import com.sun.glass.ui.Screen
+import com.sun.javafx.util.Utils
+import java.text.MessageFormat
+import javafx.animation.AnimationTimer
+import org.kordamp.ikonli.javafx.FontIcon
+import org.wycliffeassociates.otter.jvm.controls.styles.tryImportStylesheet
+import org.wycliffeassociates.otter.jvm.controls.waveform.ScrollingWaveform
+import org.wycliffeassociates.otter.jvm.utils.onChangeAndDoNow
+import org.wycliffeassociates.otter.jvm.workbookapp.ui.viewmodel.ChunkingViewModel
+import org.wycliffeassociates.otter.jvm.workbookapp.ui.viewmodel.ChunkingWizardPage
+import org.wycliffeassociates.otter.jvm.workbookapp.ui.viewmodel.SECONDS_ON_SCREEN
+import tornadofx.*
+
+class Consume : Fragment() {
+
+    val playIcon = FontIcon("mdi-play")
+    val pauseIcon = FontIcon("mdi-pause")
+
+    val vm: ChunkingViewModel by inject()
+
+    var timer: AnimationTimer? = null
+
+    override fun onDock() {
+        super.onDock()
+        tryImportStylesheet(resources.get("/css/scrolling-waveform.css"))
+        tryImportStylesheet(resources.get("/css/consume-page.css"))
+        vm.onDockConsume()
+        vm.pageProperty.set(ChunkingWizardPage.CONSUME)
+        vm.titleProperty.set(messages["consumeTitle"])
+        vm.stepProperty.set(MessageFormat.format(messages["consumeDescription"], vm.chapterTitle))
+
+        timer = object : AnimationTimer() {
+            override fun handle(currentNanoTime: Long) {
+                vm.calculatePosition()
+            }
+        }
+        timer?.start()
+    }
+
+    override fun onUndock() {
+        super.onUndock()
+        timer?.stop()
+        vm.compositeDisposable.clear()
+    }
+
+    override val root = borderpane {
+        center = ScrollingWaveform().apply {
+            positionProperty.bind(vm.positionProperty)
+
+            vm.compositeDisposable.add(
+                vm.waveform.observeOnFx().subscribe { addWaveformImage(it) }
+            )
+
+            onWaveformClicked = { vm.pause() }
+            onWaveformDragReleased = { deltaPos ->
+                val deltaFrames = pixelsToFrames(deltaPos)
+                val curFrames = vm.getLocationInFrames()
+                val duration = vm.getDurationInFrames()
+                val final = Utils.clamp(0, curFrames - deltaFrames, duration)
+                vm.seek(final)
+            }
+        }
+        bottom = hbox {
+            styleClass.addAll("consume__bottom")
+            button {
+                vm.isPlayingProperty.onChangeAndDoNow {
+                    it?.let {
+                        when (it) {
+                            true -> graphic = pauseIcon
+                            false -> graphic = playIcon
+                        }
+                    }
+                }
+                styleClass.addAll("btn", "btn--cta", "consume__btn--primary")
```

Comment:
if you have a customized button that doesn't override the existing .btn--primary selector, just use ` consume-btn ` is sufficient. 

In the case when you want to customize/override the primary button styles, then the classes should contain that css class (.btn--primary) and followed by your class: 
` .addAll(... "btn--primary", "consume__btn--primary") ` --> css will apply the former and override with whatever duplicated styles in the latter. Notice that ` btn--cta ` and ` btn--primary ` shouldn't coexist because they are usually very distinctive